tests/controllers/FeedbackControllerMock.ts

```diff
@@ -0,0 +1,41 @@
+import { Connection } from 'typeorm';
+import { UserModel } from '../../models/UserModel';
+import { DatabaseConnection, PortalState } from '../data';
+import { PublicFeedback, Feedback, Uuid, FeedbackStatus } from '../../types';
+import { ControllerFactory } from './ControllerFactory';
+
+export class FeedbackControllerMock {
```

Comment:
I mainly built it to abstract away the database connection `conn` (and things like `PortalState`) from the tests to make them simpler and easier to read

Reply:
The `beforeAll/beforeEach` already take care of the db connection, so it's
```typescript
await FeedbackControllerMock.mock(users);
```
vs.
```typescript
await new PortalState().createUsers(users).write();
```
At that point you basically have a whole wrapper for that change alone, and you'd have to do something similar for all the other controllers. Makes more sense to just keep the `PortalState` stuff in the test itself since that's what it was made for. Also as a nit, nothing's being "mocked"; we're creating real users in the database and letting the controller execute normally instead of mocking behavior for whatever components are in the controller.